batch/batch/cleanup_sidecar.py

```diff
@@ -22,8 +22,8 @@
     if copy_output_cmd is not None:
         log.info(f'copying out data')
         try:
-            pod_log = sp.check_output(copy_output_cmd, shell=True, stderr=sp.STDOUT)
-            log.info(pod_log.decode('ascii'))
+            copy_output = sp.check_output(copy_output_cmd, shell=True, stderr=sp.STDOUT)
+            log.info(copy_output.decode('ascii'))
```

Comment:
Why is this getting logged though and not saved somewhere?

Reply:
The cleanup sidecar runs a shell command, captures its output, and logs it to the cleanup sidecar's logs.

After the cleanup sidecar exits, there is one more living container: keep-alive sidecar. When batch notices cleanup sidecar has exited, it downloads the cleanup container logs and puts them in google storage.